pkg/api/http_server.go

```diff
@@ -572,6 +576,7 @@ func (hs *HTTPServer) addMiddlewaresAndStaticRoutes() {
 	m.Use(hs.apiHealthHandler)
 	m.Use(hs.metricsEndpoint)
 	m.Use(hs.pluginMetricsEndpoint)
+	hs.setupFrontendLogHandlers()
```

Comment:
Nit. Could consider using `m.Use` here since `setupFrontendLogHandlers` calls `hs.Use` multiple times now. `setupFrontendLogHandlers` would then become a HTTP handler.

Reply:
it needs to set up some singleton dependencies (source map store, rate limiter,..). It could be a HTTP handler factory though?

Reply:
Yes, you can do things like just return without writing a response:
https://github.com/grafana/grafana/blob/7104d90b394ed0e501c472abd04724cdf0edc3ad/pkg/api/plugin_metrics.go#L13-L20

No biggie really, you can leave it as is for now.

Reply:
done!